CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - alias in provider model to check if has highlighted subjects
 - `preprintDoiCreated` attribute to the `preprint` model
 - `ember-cli-clipboard` to allow copying to clipboard on more browsers
+- Functionality to move quick files to projects
+- `get-ancestor-descriptor` helper to create project/node list
+- `project-selector` component which helps user choose where to move file
```

Comment:
This should go in the [Unreleased] section

Reply:
Should also mention the changes made to the file-browser component, the file and node models, and packages moved/added.